src/lib.rs

```diff
@@ -312,6 +312,11 @@ mod tests {
             };
         }
 
+        assert_duration!("1 seconds", Interval::Seconds(1));
+        assert_duration!("24 seconds", Interval::Seconds(24));
+        assert_duration!("34 s", Interval::Seconds(34));
+        assert_duration!("34 sec", Interval::Seconds(34));
```

Comment:
These tests already pass on the latest master for me 🤔 Maybe there's something I'm missing?

Reply:
Just for sanity checks for my implementation.